cookbook/case_studies/feature_engineering/feast_integration/feast_workflow.py

```diff
@@ -0,0 +1,179 @@
+import joblib
+import os
+import numpy as np
+import pandas as pd
+from datetime import datetime
+from feast import FeatureStore, Entity, FeatureView, FileSource, ValueType, Feature, RepoConfig
+from feast.infra.online_stores.sqlite import SqliteOnlineStoreConfig
+from feast.infra.offline_stores.file import FileOfflineStoreConfig
+from flytekit import task, workflow
+from flytekit.extras.sqlite3.task import SQLite3Config, SQLite3Task
+from flytekit.types.file.file import FlyteFile
+from flytekit.types.schema import FlyteSchema
+from flytekit.types.file import JoblibSerializedFile
+from datetime import timedelta
+from feature_eng_tasks import mean_median_imputer, univariate_selection
+from sklearn.model_selection import train_test_split
+from sklearn.naive_bayes import GaussianNB
+
+
+# TODO: find a better way to define these features.
+FEAST_FEATURES = [
+    "horse_colic_stats:rectal temperature",
+    "horse_colic_stats:total protein",
+    "horse_colic_stats:peripheral pulse",
+    "horse_colic_stats:surgical lesion",
+    "horse_colic_stats:abdominal distension",
+    "horse_colic_stats:nasogastric tube",
+    "horse_colic_stats:outcome",
+    "horse_colic_stats:packed cell volume",
+    "horse_colic_stats:nasogastric reflux PH",
+]
+DATABASE_URI = "https://cdn.discordapp.com/attachments/545481172399030272/861575373783040030/horse_colic.db.zip"
+DATA_CLASS = "surgical lesion"
+
+
+def _build_feature_store(registry: FlyteFile) -> FeatureStore:
+    # TODO: comment this
+    os.environ["FEAST_S3_ENDPOINT_URL"] = os.environ["FLYTE_AWS_ENDPOINT"]
```

Comment:
i think we can do it like this
if registry.remote_path.startswith("s3://"):
     https://github.com/flyteorg/flytekit/blob/master/flytekit/extras/persistence/s3_awscli.py#L22-L30

Reply:
I have added this. @eapolinario, please check.

Reply:
If we do this we'll no longer be able to run this locally. I'm ok with that, but it's worth calling it out.

Reply:
I think we can. We may have to handle local files separately without s3, then we don't need to send the env variables.